client/driver/lxc.go

```diff
@@ -210,9 +210,20 @@ func (d *LxcDriver) Prestart(*ExecContext, *structs.Task) (*PrestartResponse, er
 
 // Start starts the LXC Driver
 func (d *LxcDriver) Start(ctx *ExecContext, task *structs.Task) (*StartResponse, error) {
+	sresp, err, errCleanup := d.StartWithCleanup(ctx, task)
+	if err != nil {
+		if cleanupErr := errCleanup(); cleanupErr != nil {
+			d.logger.Printf("[ERR] error occured:\n%v\nwhile cleaning up from error in Start: %v", cleanupErr, err)
```

Comment:
Remove the newline and `err` from this message since `err` will be logged later anyway.